python-sdk/tests/test_examples.py

```
@@ -1,9 +1,8 @@
 import json
 import os
 import importlib
-import random
 import inspect
-from typing import Optional, Sequence, cast
+from typing import Optional, Sequence
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)